Resources/Prototypes/Recipes/Construction/Graphs/utilities/ame_shielding.yml

```diff
@@ -0,0 +1,23 @@
+- type: constructionGraph
+  id: ameShielding
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: ameShielding
+          steps:
+            - prototype: AMEPart
+              doAfter: 2
```

Comment:
If this is really only for deconstruction, then you don't need this.

Reply:
Removed as requested. When I originally did this PR I was sure I was getting errors for not having complete graphs (no edge to node etc).